apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/NyttBrevModal.tsx

```diff
@@ -61,36 +65,50 @@ export const NyttBrevModal = ({ sakId }: { sakId: number }) => {
 
         <form onSubmit={handleSubmit(opprettBrev)}>
           <Modal.Body>
-            <FlexRow $spacing>
+            <VStack gap="4">
               <Select
                 error={errors?.type?.message}
                 label="Type"
                 {...register('type', {
                   required: { value: true, message: 'Feltet er påkrevd' },
                 })}
               >
-                <option value="OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_VARSELBREV">Varselbrev om aktivitetsplikt</option>
+                <option value="OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_INFORMASJON_4MND">
+                  Informasjon om aktivitetsplikt ved 4 måneder
+                </option>
                 <option value="TOMT_BREV">Manuelt brev</option>
               </Select>
-            </FlexRow>
 
-            {skjemaet.type === 'OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_VARSELBREV' && (
-              <FlexRow $spacing>
-                <Select
-                  error={errors?.aktivitetsgrad?.message}
-                  label="Aktivitetsgrad"
-                  {...register('aktivitetsgrad', {
-                    required: { value: true, message: 'Du må velge aktivitetsgrad' },
-                    validate: { notDefault: (value) => !!value },
-                  })}
-                >
-                  <option value="">Velg aktivitetsgrad</option>
-                  <option value="IKKE_I_AKTIVITET">Ikke i aktivitet</option>
-                  <option value="UNDER_50_PROSENT">Under 50%</option>
-                  <option value="OVER_50_PROSENT">Over 50%</option>
-                </Select>
-              </FlexRow>
-            )}
+              {skjemaet.type === 'OMSTILLINGSSTOENAD_AKTIVITETSPLIKT_INFORMASJON_4MND' && (
+                <>
+                  <Select
+                    error={errors?.aktivitetsgrad?.message}
+                    label="Aktivitetsgrad"
+                    {...register('aktivitetsgrad', {
+                      required: { value: true, message: 'Du må velge aktivitetsgrad' },
+                      validate: { notDefault: (value) => !!value },
+                    })}
+                  >
+                    <option value="">Velg aktivitetsgrad</option>
+                    <option value="IKKE_I_AKTIVITET">Ikke i aktivitet</option>
+                    <option value="UNDER_50_PROSENT">Under 50%</option>
+                    <option value="OVER_50_PROSENT">Over 50%</option>
```

Comment:
Mulighet for å bruke enum her for å sette value? Da slipper vi å deale med strings...